RELEASES.md

```diff
@@ -1,3 +1,117 @@
+Version 1.1.0 (June 2015)
+=========================
+
+* ~850 changes, numerous bugfixes
+
+Highlights
+----------
+
+* `std::fs` received [a number of new functions][fs] for reading
+  filesystem metadata and accessing platform-specific features.
+* The compiler contains extended explanations of many errors.  When it
+  emits such an error it also suggests using the `--explain` flag to
+  read the extended explanations, which are also [cataloged on the web
+  site][err].
+
+Language
+--------
+
+* TODO: There have been changes to dropck that should probably be
+  mentioned
+
+Libraries
+---------
+
+* The `str::split_whitespace` method splits a string on unicode
+  whitespace boundaries.
+* The [`Iterator::cloned`] method creates an iterator of `T` from
+  an iterator of `&T` by cloning each element. It is equivalent
+  to `.map(|&x| x)`.
```

Comment:
Pedantic nit: That statement isn't quite correct. `.cloned()` equivalent (<=>) to `.map(|x| x.clone())`. It only replaces/supersedes `.map(|&x| x)`.